app/controllers/concerns/authenticatable.rb

```diff
@@ -1,8 +1,8 @@
 module Authenticatable
   private
-  
+
   def sign_in(user)
-    self.current_user = user
+    create_auth_session(user)
```

Comment:
When updating authie, I noticed now that it recommends using this method over calling `self.current_user = user`